src/core/LanguageController.ts

```diff
@@ -211,13 +219,7 @@ export default class LanguageController {
         const languagePath = path.join(Config.languagesPath, address);
         const sourcePath = path.join(languagePath, 'bundle.js')
         const metaPath = path.join(languagePath, 'meta.json')
-        try {
-            fs.mkdirSync(languagePath)
-        } catch(e) {
-            console.error("Error trying to create directory", languagePath)
-            console.error("Will proceed with installing language anyway...")
-        }
-        
```

Comment:
I think this should likely still catch the error, and then just make a console.warn()

Reply:
In below `fs.writeFileSync` will throw exception if directory is not exist. try catch here seems not much helpful.

Reply:
I think the above catch is useful in the case where the language directory already exists, but something went wrong later in the install process and you want to re-install. If the `fs.mkdirSync()` actually does fail for some reason other than the directory already existing then `fs.writeFileSync()` should definitely fail (since the path to write the file is invalid) and then we can throw a gql pub sub error here.

Reply:
Improved with checking dir exist or not, pub sub can be added later.